contracts/interfaces/IThirdwebRoyalty.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
+
+import "./IThirdwebModule.sol";
+
+// Helper interfaces
+import "@openzeppelin/contracts/interfaces/IERC2981.sol";
+
+interface IThirdwebRoyalty is IThirdwebModule, IERC2981 {
+    /// @dev The recipient of who gets the royalty.
+    function royaltyRecipient() external view returns (address);
+
+    /// @dev The percentage of royalty how much royalty in basis points.
+    function royaltyBps() external view returns (uint16);
```

Comment:
all BPS should probably just be `uint16` since that's plenty to represent a max of `10000`

Reply:
**Context**

The `royaltyBps` view function is an example of a `view` function in the interface, that is a public state variable on the contract. [In ethereum, you pay gas for every storage slot you use. A slot is of 256 bits, and you can pack as many variables as you want in it.](https://mudit.blog/solidity-gas-optimization-tips/) For this reason, declaring `royaltyBps` as a standalone `uint16` without packing a storage slot with other state variables increases the contract size.

Larger contract size -> higher deployment cost + less room to maneuver for logic bug fixes or feature additions to the contract

The same correlation between 'packing' and contract size goes for parameters in a function (e.g. [constructor of `Drop721`](https://github.com/nftlabs/nftlabs-protocols/blob/v2/contracts/drop/DropERC721.sol#L120-L121)). Writing `royaltyBps` as a `uint16` state variable then has downstream effects like requiring type conversions in function bodies.

**Re: for changes like in the snippet**

The rationale for setting `royaltyBps` as `uint16` is clear ✌🏽 However, it doesn't let us eliminate this sanity check already in place in [`setRoyaltyInfo`](https://github.com/nftlabs/nftlabs-protocols/blob/v2/contracts/drop/DropERC721.sol#L349) since a `uint16` can exceed `10_000` anyway. As a result, turning `royaltyBps` to a `uint16` only increases the contract size without improving any sanity checks for `royaltyBps`.